demos/formula_recognition_demo/python/README.md

```diff
@@ -64,6 +64,12 @@ Ubuntu:
 `apt-get update && apt-get install texlive`
 MacOS:
 `brew install texlive`
+
+If you face the `RuntimeError: dvipng is not installed` error, you need to install this library. For Linux you can do it via
+`apt-get update && apt-get install dvipng`.
+You might also face the problem of missing `standalone.cls` file, it could be fixed with installation `texlive-latex-extra` package. For Linux it can be done using this command :
```

Comment:
```suggestion
You might also face the missing `standalone.cls` file problem, which could be fixed with the installation of `texlive-latex-extra` package. For Linux, it can be done using this command:
```